internal/configuration/schema/notifier.go

```diff
@@ -23,6 +23,7 @@ type SMTPNotifierConfiguration struct {
 	StartupCheckAddress mail.Address  `koanf:"startup_check_address"`
 	DisableRequireTLS   bool          `koanf:"disable_require_tls"`
 	DisableHTMLEmails   bool          `koanf:"disable_html_emails"`
+	DisableStartTLS     bool          `koanf:"disable_start_tls"`
```

Comment:
```suggestion
	DisableStartTLS     bool          `koanf:"disable_starttls"`
```

Really unsure which is better, I'm just leaving this here for discussion. We also need to add documentation to this, similar to my log comment.

Reply:
applied suggested change